types.go

```diff
@@ -50,6 +50,7 @@ type Settings struct {
 	FilterableAttributes []string            `json:"filterableAttributes,omitempty"`
 	SortableAttributes   []string            `json:"sortableAttributes,omitempty"`
 	TypoTolerance        *TypoTolerance      `json:"typoTolerance,omitempty"`
+	Pagination           *Pagination         `json:"pagination"`
```

Comment:
```suggestion
	Pagination           *Pagination         `json:"pagination,omitempty"`
```
We may need the `omitempty` to update the Setting without modifying the pagination.